### PR TITLE
Feat/jiva 98/cost anomaly controls

### PR DESCRIPTION
**Pull request by dgavrischuk**

### CODE REVIEW

File: AWS/anomaly_monitor/providers.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+terraform {
+  experiments = [module_variable_optional_attrs]
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+provider "aws" {
```
Comment (AlexBMes) [inline AWS/anomaly_monitor/providers.tf:10]: Provider block probably shouldn't be included (see [comment in other PR](https://github.com/mesoform/Multi-Cloud-Platform-Foundations/pull/14#discussion_r977500799))

File: AWS/anomaly_monitor/vars.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+variable "region" {
+  default = "eu-central-1"
+}
+variable "subscription_email" {
+  default = "dmytro.gavryschuk@mesoform.com"
```
Comment (AlexBMes) [inline AWS/anomaly_monitor/vars.tf:5]: Remove default variable with personal info

**Review by AlexBMes — CHANGES_REQUESTED** [review]
See inline comments

File: AWS/snapshot_recovery/script/restore.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```
Comment (gbmeuk) [inline AWS/snapshot_recovery/script/restore.sh:1]: remove this script

**Pull request closed without merging**